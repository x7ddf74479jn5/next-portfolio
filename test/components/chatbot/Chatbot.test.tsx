import Chatbot from "src/components/chatbot/Chatbot2";
import type { RenderResult } from "test/test-utils";
import { render } from "test/test-utils";

describe("Chatbot", () => {
  let renderResult: RenderResult;

  beforeEach(() => {
    renderResult = render(<Chatbot />);
  });

  it("matches snapshot", () => {
    expect(renderResult.asFragment()).toMatchSnapshot();
  });
});
