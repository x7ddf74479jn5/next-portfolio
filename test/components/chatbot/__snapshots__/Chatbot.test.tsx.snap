// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chatbot matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="section"
  >
    <div
      class="box"
    >
      <ul
        class="MuiList-root makeStyles-chats-1 MuiList-padding"
        id="scroll-area"
      >
        <li
          class="MuiListItem-root chatRow MuiListItem-gutters"
        >
          <div
            class="MuiListItemAvatar-root"
          >
            <div
              class="MuiAvatar-root MuiAvatar-circle"
            >
              <img
                alt="icon"
                class="MuiAvatar-img"
                src="/img/icons/pandashark_180-180.png"
              />
            </div>
          </div>
          <div
            class="chatBubble"
          >
            こんにちは！ご用件はなんでしょうか？
          </div>
        </li>
      </ul>
      <div
        class="gridAnswer"
      >
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            仕事を依頼したい
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            あなたについて知りたい
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            献立を提案してほしい
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined makeStyles-button-2"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            その他の問い合わせがしたい
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
